src/components/BackgroundGradient/BackgroundGradient.tsx

```diff
@@ -1,22 +1,42 @@
 'use client';
-import type { CSSObject } from '@mui/material';
+import { Box, type CSSObject } from '@mui/material';
 import {
   BackgroundGradientBottomLeft,
   BackgroundGradientBottomRight,
   BackgroundGradientContainer,
   BackgroundGradientTopCenter,
 } from '.';
+import { usePathname } from 'next/navigation';
 
 interface BackgroundGradientProps {
   styles?: CSSObject;
 }
 
 export const BackgroundGradient = ({ styles }: BackgroundGradientProps) => {
-  return (
+  const pathname = usePathname();
+
+  return !pathname.includes('memecoins') ? (
     <BackgroundGradientContainer sx={styles}>
       <BackgroundGradientBottomLeft />
       <BackgroundGradientBottomRight />
       <BackgroundGradientTopCenter />
     </BackgroundGradientContainer>
+  ) : (
+    <Box
```

Comment:
I would recommend to use https://nextjs.org/docs/pages/api-reference/components/image to enhance the lazy loading of it